dolphinscheduler-task-plugin/dolphinscheduler-task-java/src/main/java/org/apache/dolphinscheduler/plugin/task/java/JavaTask.java

```diff
@@ -183,9 +184,8 @@ protected String buildJarCommand() {
         StringBuilder builder = new StringBuilder();
         builder.append(getJavaCommandPath())
                 .append("java").append(" ")
-                .append(buildResourcePath()).append(" ")
+                .append(buildExtDirs()).append(" ")
```

Comment:
Can we refactor this, use `-cp` rather than `-jar`.

```
$JAVA_HOME/bin/java $JVM_ARGS \
  -cp "$taskInstanceWorkingPath" \
  Main.class
```

Reply:
If you use -jar, you need to specify the entry class, but the page cannot specify the entry class on the configuration page when you select the jar type

Reply:
Yes，I just find we cannot set entry class......
So we only support fat jar, this is sad.
So the final command should look like 
```
$JAVA_HOME/bin/java -jar xx.jar $JVM_ARGS \
  -cp xx
```
Can we remove `-Djava.ext.dirs`, it seems not a good idea to use this param.

Reply:
It seems that -jar cannot work with -cp together,-jar cannot get -cp's libraries,Is your shell worked success？have you test your shell script?

Reply:
> If you use -cp, you need to specify the entry class, but the page cannot specify the entry class on the configuration page when you select the jar type

Reply:
> It seems that -jar cannot work with -cp together,-jar cannot get -cp's libraries,Is your shell worked success？have you test your shell script?

Yes, -jar can not work with -cp.

Once we use -jar then the jar should be a fat jar, the better way is to add ext jar by set Class-Path at `MANIFEST.MF`.

So Can we use -cp with entry class(add a new param) rather than -jar.